mlflow/server/auth/config.py

```diff
@@ -27,4 +28,5 @@ def read_auth_config() -> AuthConfig:
         database_uri=config["mlflow"]["database_uri"],
         admin_username=config["mlflow"]["admin_username"],
         admin_password=config["mlflow"]["admin_password"],
+        authorization_function=config["mlflow"]["authorization_function"],
```

Comment:
I think this line throws if the config doesn't have `authorization_function`. Can we deafult to `mlflow.server.auth:authenticate_request_basic_auth` if it's missing?

Reply:
Sure.